app/views/shared/_header.html.erb

```diff
@@ -16,8 +16,17 @@
           </li>
 
           <li>
-            <%= link_to decorated_current_user.github_url, class: 'tooltipped tooltipped-s', 'aria-label' => 'View GitHub profile' do %>
-              <%= image_tag decorated_current_user.avatar_url(40), class: 'avatar', alt: "@#{decorated_current_user.login}", height: 20, width: 20 %>
+            <% if current_user != true_user %>
```

Comment:
that seems more clear to me as well, I'm going to go ahead and make that change.